analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -529,7 +529,7 @@ class MavenSupport(private val workspaceReader: WorkspaceReader) {
             val localRepositories = allRepositories - remoteRepositories
             if (localRepositories.isNotEmpty()) {
                 // No need to use curly-braces-syntax for logging here as the log level check is already done above.
-                logger.debug { "Ignoring local repositories $localRepositories." }
+                logger.debug("Ignoring local repositories $localRepositories.")
```

Comment:
What's the advantage of not using curly braces?

Reply:
Avoiding a superfluous check if the log level is set to debug.

Reply:
I believe such micro optimizations should not be valued that high, and consistency preferred in this case. Which would also allow removing that comment in line 531.

Reply:
> Which check actually is superflous?

The `logger.delegate.isDebugEnabled` condition duplicates the check done in the implementation if `debug {}`.

> I believe such micro optimizations should not be valued that high

I'm not valuing it high, I'm just changing the code to match the documentation.

> consistency preferred in this case.

There are also other code locations where is "micro-optimization" is performed, and I prefer to stay consistent to these.

Reply:
> I'm not valuing it high

I believe my point was clear, e.g. I believe it's inappropriate to do such kind of optimizations in the given context.

> There are also other code locations where is "micro-optimization" is performed, and I prefer to stay consistent to these.

Would you mind sharing why?

I'll share my reason as well: keep it simple.